lib/twitter/processTweetData.js

```
@@ -0,0 +1,59 @@
+const processTweetData = (tweet) => {
+  try {
+    if (!tweet || !tweet.id) return null;
+
+    let timestamp = tweet.timestamp;
+    if (!timestamp) {
+      timestamp = tweet.timeParsed?.getTime();
+    }
+
+    if (!timestamp) return null;
+    if (timestamp < 1e12) timestamp *= 1000;
+    if (isNaN(timestamp) || timestamp <= 0) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)